scanner/src/components/DataRow/index.svelte

```diff
@@ -1,25 +1,38 @@
 <script lang="ts">
   export let label: string
   export let value: string | string[]
+  export let link: boolean = false
 </script>
 
 <div class="row">
   <div class="label">
     {label}:
   </div>
-  <div class="value">
-    {#if typeof value === "string"}
-      {value.trim() || "-"}
-    {:else if value === null || value === undefined}
-      -
-    {:else if value.length === 0}
-      <div>-</div>
-    {:else}
-      {#each value as v}
-        <div>{v.trim() || "-"}</div>
-      {/each}
-    {/if}
-  </div>
+  {#if link || (typeof value === "string" && value.startsWith("https://"))}
+    <div class="value">
+      {#if typeof value === "string"}
+        <a class="link" href={value.trim()} target="_blank" rel="noreferrer">{value.trim()}</a>
```

Comment:
I don't believe you should be trimming values. Why is this done? In either and both cases?